### PR TITLE
[italian translation] First commit; welcome page

### PR DESCRIPTION
**Pull request by lodo1995**
First effort in translating the tour in italian. It will probably need some corrections.
I don't know many people in the community, so I'm not sure who could review this. Maybe @llucax ?

### CODE REVIEW

**Comment by wilzbach** [comment]
> I don't know many people in the community, so I'm not sure who could review this. Maybe @llucax ?

Someone created the D map a while ago:

https://www.google.com/maps/d/edit?mid=1euIpoA6stFHlmIk1m9qbVHwP_64

Italy is mostly empty except for @trikko ;-)

File: public/content/it/welcome/welcome-to-d.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+### Informazioni sul tour
+
+Ogni sezione è accompagnata da un esempio di codice che può essere modificato e testato
+per sperimentare le caratteristiche del linguaggio D.
+Premi il pulsante esegui (o `Ctrl-Invio`) per compilarlo ed eseguirlo.
+
+### Contribuire al tour
+
+Questo tour è [open source](https://github.com/stonemaster/dlang-tour/tree/master/public/content/en)
```
Comment (wilzbach) [inline public/content/it/welcome/welcome-to-d.md]: I think we should drop the latter `en` bit, #420 changes this for `en` ;-)

File: public/content/it/index.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+title: Il tour di D
```
Comment (trikko) [inline public/content/it/index.yml]: It sounds bad to me "di D".  "di" and "D" are read in the same way in italian. Maybe "Il tour del linguaggio D?"

File: public/content/it/welcome/welcome-to-d.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Questo tour ti fornirà le basi di questo  __potente__ ed __espressivo__
+linguaggio, che compila in __efficiente__ codice __nativo__.
+
+{{/dmanmobile}}
+
+### Cos'è D?
+
+D rappresenta il culmine di  _decenni di esperienza nell'implementazione di compilatori_
+per molti linguaggi differenti e dispone di un gran numero di
+[feature uniche](http://dlang.org/overview.html):
```
Comment (trikko) [inline public/content/it/welcome/welcome-to-d.md]: "feature" => "Caratteristiche"?

File: public/content/it/welcome/welcome-to-d.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- interfacciamento diretto con le API del sistema operativo e con l'hardware
+- tempi di compilazione estremamente rapidi
+- programmazione memory-safe (SafeD)
+- codice _mantenibile_ e _facile da comprendere_
+- bassa curva di apprendimento (sintassi C-like, simile a Java e molti altri linguaggi)
+- compatibile con il codice C a livello di ABI
+- linguaggio multi-paradigma (programmazione imperativa, strutturata, orientata agli oggetti, generica, funzionale e persino codice assembly integrato)
+- costrutti di prevenzione degli errori (asserzioni, contratti, unit testing)
+
+... e molte altre [features](http://dlang.org/overview.html).
```
Comment (trikko) [inline public/content/it/welcome/welcome-to-d.md]: Features => caratteristiche (anyway: in italian foreing words are always singular. We don't say "i computers")

File: public/content/it/welcome/welcome-to-d.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# Benvenuto al tour di D
+
+Benvenuto al tour interattivo del *Linguaggio di Programmazione D*.
```
Comment (trikko) [inline public/content/it/welcome/welcome-to-d.md]: "Benvenuto nel"

File: public/content/it/index.yml (hunk 2 above)
Comment (lodo1995) [inline public/content/it/index.yml]: I'm not a great fan of long titles, but you are right that the current one sounds quite bad, so I'll change it. Thanks.

File: public/content/it/welcome/welcome-to-d.md (hunk 3 above)
Comment (lodo1995) [inline public/content/it/welcome/welcome-to-d.md]: Sounds good.

File: public/content/it/welcome/welcome-to-d.md (hunk 5 above)
Comment (lodo1995) [inline public/content/it/welcome/welcome-to-d.md]: I'm not sure about this. Switching to italian for lack of good translation.
Il complemento di stato in luogo si può esprimere sia con "a" che con "in" (in casi differenti) e non sono sicuro di quale sia corretto qui. Di solito diciamo cose tipo "al tour de France", "al gran tour". E' difficile trovare buoni esempi perché in effetti "tour" è una parola francese. "In" è solitamente approssimabile a "dentro" e "a" a "presso". A me personalmente "nel" piace di più in questo contesto, ma se hai informazioni più accurate su quale sia corretto, ben venga.
TL;DR Italian is a mess even for native speakers.

File: public/content/it/welcome/welcome-to-d.md (hunk 4 above)
Comment (lodo1995) [inline public/content/it/welcome/welcome-to-d.md]: "Caratteristiche" is quite a neutral term (in the previous spot, it was made positive by "unique"), while "features" is inherently positive. So what about a stronger word like "peculiarità" or "qualità"?

Comment (trikko) [inline public/content/it/welcome/welcome-to-d.md]: Yes or we can add an adjective here too "interessanti" "uniche"

File: public/content/it/welcome/welcome-to-d.md (hunk 5 above)
Comment (trikko) [inline public/content/it/welcome/welcome-to-d.md]: Concordo con l'analisi. Io propenderei per il "nel" perché è più immersivo. Cioè entri nel tour in questo caso, ti addentri. "Al tour" mi sembra qualcosa del tipo "sei arrivato all'inizio del tour" (di certo nessuno direbbe "nell'inizio del tour"). Magari direi "alla prima fase del tour" / "alla seconda" piuttosto che "nella prima" / "nella seconda". Ma sono leggere sfumature di significato indubbiamente :)

Forse un esempio che aiuta a capire cosa intendo è: "bevenuto al bosco incantato" / "benvenuto nel bosco intantato". Nel secondo ci sei immerso dentro. Mi sembra più coinvolgente essere dentro il tour.

Comment (lodo1995) [inline public/content/it/welcome/welcome-to-d.md]: You have a point. I'll switch to "Benvenuto nel" as it gives a better feeling to the reader. Thanks for the feedback.

**Comment by lodo1995** [comment]
Changed some wording based on feedback from @trikko and @wilzbach .

**Comment by wilzbach** [comment]
> Changed some wording based on feedback from @trikko and @wilzbach .

Nice!
@trikko thanks a lot for taking your time to review this. I will merge as soon as you give your okay ;-)

**Comment by llucax** [comment]
> Maybe @llucax ?

Haha, despite my last name (and that I actually also have Italian citizenship), I'm Argentinian and never learned proper Italian. I can understand (mostly), but I think I'm not able to do a review :)

**Comment by lodo1995** [comment]
@llucax ah, sorry for bothering you.
I should better check the informations before making assumptions. :)

**Comment by lodo1995** [comment]
Ping @trikko can you give your final ok for this to get merged?

**Comment by trikko** [comment]
Yes it's ok for me (sorry for late reply i'm out for some days, reading from my mobile phone)

**Comment by lodo1995** [comment]
@trikko thank you very much.

**Comment by wilzbach** [comment]
thanks @lodo1995 and @trikko :)

@lodo1995 the tour is automatically deployed, so in a few minutes, it should be up at http://tour.dlang.org/tour/it/welcome/welcome-to-d

**Comment by lodo1995** [comment]
@wilzbach that's a great thing!
When it will have some more pages (I'm working on them in my little free time) I'll ask some friends of mine (italian programmers that don't know D) to provide some un-biased feedback on it.
I guess it's accessible only via direct URL, not linked somewhere (that would be very bad, until it's finished).

**Comment by wilzbach** [comment]
> @wilzbach that's a great thing!

Full credits go to @stonemaster for setting this nice process up ;-)

> When it will have some more pages (I'm working on them in my little free time) I'll ask some friends of mine (italian programmers that don't know D) to provide some un-biased feedback on it.

Looking forward to it :)
Btw good luck with your final week & submission!

> I guess it's accessible only via direct URL, not linked somewhere (that would be very bad, until it's finished).

Yep, no worries ;-)

**Comment by lodo1995** [comment]
> Btw good luck with your final week & submission!

Thank you. Good luck to you too!
